packages/symfony-toggle/composer.json

```diff
@@ -15,17 +15,18 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
-        "symfony/framework-bundle": "~5.0",
         "pheature/toggle-core": "@dev",
         "pheature/toggle-crud": "@dev",
         "pheature/toggle-crud-psr7-api": "@dev",
         "pheature/toggle-crud-psr11-factories": "@dev",
         "pheature/toggle-model": "@dev",
+        "symfony/framework-bundle": "~5.0",
         "symfony/psr-http-message-bridge": "^2.1"
     },
     "require-dev": {
         "doctrine/dbal": ">=2.5 || ^3.0.0",
         "infection/infection": "^0.21.0",
+        "pheature/inmemory-toggle": "@dev",
```

Comment:
I think this dependency it's not needed here. What do you think?

Reply:
Yes, is a dev-dependency, also I put here `toggle-crud` and `toggler-crud-psr7-api` packages, and remove `doctrine/dbal` and `symfony/psr-http-message-bridge` because they are a second-level dependency. Also, I added our packages to the composer's "suggest" option with a small purpose description.